### PR TITLE
Fix agent rpm builds

### PR DESCRIPTION
**Pull request by portante**

### CODE REVIEW

File: agent/rpm/Makefile [hunk 1]
```diff
@@ -53,8 +53,7 @@ patches:
 .PHONY: tarball
 tarball:
 	mkdir -p ${RPMSRC}
-	git submodule init
-	git submodule update
+	(cd ${AGENT}/../; git submodule init; git submodule update)
```
Comment (npalaska) [inline agent/rpm/Makefile:56]: just a question, does this change the working directory and do we need to cd back to the rpm directory?

**Review by zulcss — COMMENTED** [review]
lgtm

File: agent/rpm/Makefile (hunk 1 above)
Comment (portante) [reply agent/rpm/Makefile:56]: The closing parentheses should prevent the changing of the default working directory from escaping just that command.

**Comment by portante** [comment]
> lgtm

Can you officially approve it to allow a merge?

**Review by zulcss — APPROVED** [review]

File: agent/rpm/Makefile (hunk 1 above)
Comment (ndokos) [inline agent/rpm/Makefile:56]: Why does it matter? You can do `git submodule init/update` from anywhere in the tree.

**Review by ndokos — CHANGES_REQUESTED** [review]

File: agent/rpm/Makefile (hunk 1 above)
Comment (portante) [reply agent/rpm/Makefile:56]: Apparently not on a RHEL 7 box ...

**Review by npalaska — APPROVED** [review]

**Review by dbutenhof — APPROVED** [review]

**Review by Maxusmusti — APPROVED** [review]

File: agent/rpm/Makefile (hunk 1 above)
Comment (ndokos) [reply agent/rpm/Makefile:56]: This is a shell mechanism: executing a command inside parens does it in a subshell, so changing the directory in the subshell does not affect the parent shell at all.

Comment (ndokos) [reply agent/rpm/Makefile:56]: I am skeptical: I'll go test.

**Review by ndokos — APPROVED** [review]

**Pull request merged**